backend/logistica_servicio/gestor_entregas/tests/test_repositorio_entregas.py

```
@@ -0,0 +1,136 @@
+import unittest
+import datetime
+from infraestructura.repositorio_entregas import RepositorioEntrega, Session, RepositorioDetalleEntrega
+from infraestructura.modelos import Entrega, DetalleEntrega, Base
```

Ruff (0.8.2)

`infraestructura.modelos.Base` imported but unused

Remove unused import: `infraestructura.modelos.Base`

(F401)